kong/clustering/services/sync/strategies/postgres.lua

```diff
@@ -2,9 +2,15 @@ local _M = {}
 local _MT = { __index = _M }
 
 
+local fmt = string.format
 local ngx_null = ngx.null
 
 
+-- version string should be greater than any hex string(ngx.md5)
+-- e.g.: "v02_0000" > "FFFFFFFF"
+local VERSION_FMT = "v02_%028d"
```

Comment:
```suggestion
local VERSION_FMT = "v02_%028x"
```

We should not use octets, use hex instead.